src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -179,11 +181,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ItemGroup>
 
     <!--
-        Not using SkipUnchangedFiles="true" because the application may want to change
-        one of these files and not have an incremental build replace it.
-        -->
+      Use SkipUnchangedFiles for the Copy task to prevent unnecessary copying, but
+      unlike "PreserveNewest", copy anything that differs by file size.
```

Comment:
This is file size or timestamp, right?

Reply:
I'll update to make that more clear.  That's correct, missing "in addition to" sentiment.

Reply:
How about:

```xml
<!-- 
    PreserveNewest means that we will only copy the source to the destination if the source is newer.
    SkipUnchangedFiles is not used for that purpose because it will copy if the source and destination
    differ by size.  Instead, this target uses inputs and outputs to only copy when the source is newer.
    -->
```

Reply:
How about:

```xml
 <!--
    Use SkipUnchangedFiles to prevent unnecessary file copies. The copy will occur if the
    destination doesn't exist, the source is newer than the destination, or if the source and
    destination differ by file size.
    -->
```